PocketCastsTests/UnitTests.xctestplan

```diff
@@ -52,6 +52,10 @@
   },
   "testTargets" : [
     {
+      "skippedTests" : [
+        "IAPHelperTests",
```

Comment:
Isn't this first one enough to disable the whole file?

Reply:
I used the test plan to skip it in my first commit, but it still ran 🤷‍♂️ .

Reply:
Weird, it skipped for me locally. Better to be safe than sorry, I guess! 😅